DokuwikiXMLExport.php

```diff
@@ -72,14 +84,20 @@ private function getPageIds()
     {
         $indexer = new Doku_Indexer();
         $pagesAndDeletedPages = $indexer->getPages();
-        // Get all pages that do have a description
+
+        // Get all pages that do have a description and a title set
         $pagesAndDeletedPages = array_filter($pagesAndDeletedPages, function ($page, $k) {
-            return (p_get_metadata($page)['description'] !== '' && p_get_metadata($page)['title'] !== '');
+            $pageDescription = p_get_metadata($page)['description'];
```

Comment:
I'd rather turn the variables into booleans, eg.

```php
$pageDescriptionIsNotEmpty = !empty(p_get_metadata($page)['description']);
$pageTitleIsNotEmpty = !empty(p_get_metadata($page)['title']);
return $pageDescriptionIsNotEmpty && $pageTitleIsNotEmpty;
```
(Hope I got the logic right)

This makes it easier to read IMHO.

Reply:
I thought about this but I wasn't sure if it makes sense to save the booleans in their variables. Will implement, because the readability is so much better.

Reply:
:white_check_mark: